packages/api/src/mappings/google/activity.ts

```diff
@@ -99,22 +208,48 @@ export const mapToActivity = (
     });
   }
 
-  if (googleSessions) {
-    const activitySessions = googleSessions.session.filter(
-      session => session.activityType !== sessionSleepType
-    );
-
-    const activityLogs = activitySessions.map(session => {
-      return {
-        metadata,
-        name: GoogleActivityTypes[session.activityType],
-        start_time: dayjs(Number(session.startTimeMillis)).toISOString(),
-        end_time: dayjs(Number(session.endTimeMillis)).toISOString(),
-      };
-    });
+  return activity;
+};
 
-    activity.activity_logs = activityLogs;
+function nanoTimeString(startTime: string | undefined): number {
+  if (startTime) {
+    const dateObject = new Date(startTime);
+    const timeInNanoseconds = dateObject.getTime() * 1e6;
```

Comment:
`1e6` is a magic number that's not useful

use our convert lib? `convert(dateObject.getTime()).from("ms").to("ns")`